src/lib.rs

```diff
@@ -428,6 +428,22 @@ impl Build {
         self
     }
 
+    /// removes a flag from self.flags
+    ///
+    /// # example
+    /// ```no_run
+    /// cc::Build::new()
+    ///     .file("src/foo.c")
+    ///     .flag("unwanted_flag")
+    ///     .remove_flag("unwanted_flag")
+    ///     .compile("bar");
+    /// ```
+
+    pub fn remove_flag(&mut self, flag: &str) -> &mut Build {
+        self.flags.retain(|other_flag| other_flag == &flag.into());
```

Comment:
Please don't use `flag.into()`, it will convert `&str` to `Arc<str>` and thus creates an unnecessary heap allocation:

```suggestion
        self.flags.retain(|other_flag| &**other_flag == flag);
```